### PR TITLE
Make JSRuntime.Current non-public

### PR DESCRIPTION
**Pull request by pranavkm**
Fixes https://github.com/aspnet/AspNetCore/issues/6828

### CODE REVIEW

File: src/JSInterop/Microsoft.JSInterop/src/JSRuntime.cs [hunk 1]
```diff
@@ -11,13 +11,9 @@ namespace Microsoft.JSInterop
...
     {
-        private static AsyncLocal<IJSRuntime> _currentJSRuntime
-            = new AsyncLocal<IJSRuntime>();
+        private static readonly AsyncLocal<IJSRuntime> _currentJSRuntime = new AsyncLocal<IJSRuntime>();
 
-        /// <summary>
-        /// Gets the current <see cref="IJSRuntime"/>, if any.
-        /// </summary>
-        public static IJSRuntime Current => _currentJSRuntime.Value;
+        internal static IJSRuntime Current => _currentJSRuntime.Value;
```
Comment (rynowak) [inline src/JSInterop/Microsoft.JSInterop/src/JSRuntime.cs]: It says remove. But from looking at it, I can see that it is not removed.

Comment (SteveSandersonMS) [reply src/JSInterop/Microsoft.JSInterop/src/JSRuntime.cs]: Indeed. Could the entire `JSRuntime.cs` file be removed?

Comment (pranavkm) [reply src/JSInterop/Microsoft.JSInterop/src/JSRuntime.cs]: Not in any trivial way. It's used from a static context by `DotnetNetDispatcher` to notify of completion:

* https://github.com/aspnet/Extensions/blob/master/src/JSInterop/Microsoft.JSInterop/src/DotNetDispatcher.cs#L99
* https://github.com/aspnet/Extensions/blob/master/src/JSInterop/Microsoft.JSInterop/src/DotNetDispatcher.cs#L192-L193

In addition, there's also some pretty gnarly coupling of the runtime \ SimpleJSON's serialization and object tracking. I was hoping not to mess around with the code here if it's meant to be cleaned up soon.

**Comment by pranavkm** [comment]
Bump

**Pull request merged**